### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by neha-viswanathan**
All the existing documentation are now under `docs/legacy` directory. The CLI documentation can be found under `docs`, along with `README.md`.

### CODE REVIEW

File: docs/cluster.md [hunk 1]
```diff
@@ -5,12 +5,13 @@ The `amp cluster` command manages all cluster-related operations for AMP.
...
 
-    Usage:	amp cluster [OPTIONS] COMMAND [ARGS...]
+    Usage:	amp cluster COMMAND
 
     Cluster management operations
 
     Options:
-          --help   Print usage
+      -h, --help            Print usage
+      -s, --server string   Specify server (host:port)
```
Comment (ndegory) [inline docs/cluster.md:14]: should be `-s, --server string   Specify server (default "cloud.atomiq.io:50101")`
this is true for all commands, and should be implemented in the CLI.

**Review by ndegory — CHANGES_REQUESTED** [review]

File: docs/cluster.md (hunk 1 above)
Comment (bquenin) [reply docs/cluster.md:14]: Actually, `-s` is a root command option that should only be displayed when using `amp --help` and not be repeated on all commands.

Comment (ndegory) [reply docs/cluster.md:14]: Also, we'd like to keep a few md files from legacy, and update the README.md. Some links are broken now.

Comment (neha-viswanathan) [reply docs/cluster.md:14]: The docs will be updated soon. For now, all the old files are under `legacy`.

**Review by ndegory — APPROVED** [review]

**Pull request merged**